Sources/FileManager+Testable.swift

```diff
@@ -54,7 +72,7 @@ public extension FileManager {
         }
 
         guard let directoryUrl = directoryUrl else {
-            Log.error(label: LOG_TAG, "Could not compute the directory URL for removal.")
+            Log.error(label: LOG_TAG, "Could not compute the directory URL for \(directoryName) for removal.")
```

Comment:
Github isn't letting me mark the name of the function as the line. But, can we change the name of the function. Technically, this function isn't clearing the "cache", it is clearing the fileSystemStorage which is actually found in the Library directory (not cache). As far as the implementation, have you tested that this actually clears all of the directories? The implementation in the Core helper is different currently. See [here](https://github.com/adobe/aepsdk-core-ios/blob/main/AEPServices/Mocks/DatastoreUtilities.swift#L43).